src/api/ans.ts

```diff
@@ -161,4 +165,51 @@ export class ANS {
   async renewDomain(args: { sender: Account; name: string; years?: 1 }): Promise<InputSingleSignerTransaction> {
     return renewDomain({ aptosConfig: this.config, ...args });
   }
+
+  /**
+   * Fetches a single name from the indexer
+   * @param args.name - A string of the name to retrieve, e.g. "test.aptos.apt"
+   * or "test.apt" or "test". Can be inclusive or exclusive of the .apt suffix.
```

Comment:
we use different subdomain/domain example names in different function documentation, I think it would make it clearer if we use the same examples

Reply:
Excellent insight! I'll make the change.